### PR TITLE
Undertow configuration

### PR DESCRIPTION
**Pull request by spriadka**

### CODE REVIEW

File: start-wildfly.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
 CONTAINER=hal-wildfly
+NETWORK=testnetwork
 
-docker run -p 8080:8080 -p 9990:9990 -d --name ${CONTAINER} halconsole/hal-wildfly /opt/jboss/wildfly/bin/standalone.sh -c standalone-full-ha-insecure.xml -b 0.0.0.0 -bmanagement 0.0.0.0
+docker network create --subnet 172.80.0.0/28 testnetwork
+docker run -p 8080:8080 -p 9990:9990 -d --name ${CONTAINER} halconsole/hal-wildfly /opt/jboss/wildfly/bin/standalone.sh -c standalone-full-ha-insecure.xml -b 172.80.0.2 -bmanagement 172.80.0.2
```
Comment (spriadka) [inline start-wildfly.sh]: This is due to``ModClusterFilterTest`` and ``ModClusterFilterAttributesTest``, as they require the server not to be bound to the `0.0.0.0` address

Comment (zebrik) [reply start-wildfly.sh]: Can you please add this comment to the script?

**Comment by zebrik** [comment]
@spriadka please resolve the conflict (sorry), add requested comment a feel free to merge.

**Comment by spriadka** [comment]
@zebrik Conflicts were resolved, comment was added to the script. Merging now

**Pull request merged**